packages/babel-plugin-transform-es2015-modules-commonjs/src/index.js

```diff
@@ -9,8 +9,13 @@ let buildRequire = template(`
 `);
 
 let buildExportsModuleDeclaration = template(`
-  Object.defineProperty(exports, "__esModule", {
-    value: true
+  Object.defineProperties(exports, {
+    __esModule: {
+      value: true
+    },
+    __esExports: {
+      value: $0
```

Comment:
Does this need to be stored on `exports`. Couldn't it be a local variable instead?